tests/Doctrine/Tests/Inflector/Rules/Dutch/DutchFunctionalTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Inflector\Rules\Dutch;
+
+use Doctrine\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+use Doctrine\Inflector\Language;
+use Doctrine\Tests\Inflector\Rules\LanguageFunctionalTest;
+
+class DutchFunctionalTest extends LanguageFunctionalTest
+{
+    /**
+     * @return string[][]
+     */
+    public function dataSampleWords() : array
+    {
+        return [
+            ['weerman', 'weermannen'],
+            ['ei', 'eieren'],
+            ['rij', 'rijen'],
+            ['mogelijkheid', 'mogelijkheden'],
+            ['adres', 'adressen'],              // @todo fail
```

Comment:
```suggestion
            ['adres', 'adressen'],
```

Reply:
i do not succeed all tests yet ....

Reply:
Oh sorry, didn't notice the tests were failing

Reply:
the code passes the tests now.